engine/envelope/envelope.go

```diff
@@ -78,17 +85,19 @@ func (e *Envelope) NewTimeout(
 	id string,
 	m dogma.Message,
 	t time.Time,
+	s time.Time,
```

Comment:
What is `t` and `s`? Due to there being two time values, should the docs describe these a bit?

Reply:
Yeah, you're right.

I think I'll rename the struct fields too:

- `Time` -> `CreatedAt`
- `TimeoutTime` -> `ScheduledFor`

Or something like that.

Reply:
That sounds much clearer.